pkg/okteto/auth.go

```diff
@@ -97,36 +99,44 @@ func Auth(ctx context.Context, code, url string) (*User, error) {
 func (c *OktetoClient) queryUser(ctx context.Context) (*User, error) {
 	var query struct {
 		User struct {
-			Id              graphql.String
-			Name            graphql.String
-			Email           graphql.String
-			ExternalID      graphql.String `graphql:"externalID"`
-			Token           graphql.String
-			New             graphql.Boolean
-			Registry        graphql.String
-			Buildkit        graphql.String
-			Certificate     graphql.String
-			GlobalNamespace graphql.String `graphql:"globalNamespace"`
+			Id               graphql.String
+			Name             graphql.String
+			Email            graphql.String
+			ExternalID       graphql.String `graphql:"externalID"`
+			Token            graphql.String
+			New              graphql.Boolean
+			Registry         graphql.String
+			Buildkit         graphql.String
+			Certificate      graphql.String
+			GlobalNamespace  graphql.String  `graphql:"globalNamespace"`
+			TelemetryEnabled graphql.Boolean `graphql:"telemetryEnabled"`
 		} `graphql:"user"`
 	}
 	err := c.client.Query(ctx, &query, nil)
 	if err != nil {
 		if strings.Contains(err.Error(), "Cannot query field \"globalNamespace\" on type \"me\"") {
 			return c.deprecatedQueryUser(ctx)
 		}
+		if strings.Contains(err.Error(), "Cannot query field \"telemetryEnabled\" on type \"me\"") {
+			return c.deprecatedQueryUser(ctx)
```

Comment:
In this case, the default value of `telemetryEnabled` should be `true`, to keep the current behavior.

Reply:
Changed the user.TelemetryEnabled type to string, in case the query/mutation does not return this data, empty string will be saved in the user, and when saving this value in the context, if this is empty string, the default value true will be saved into the context